SmartEVSE2.X/EVSE.c

```diff
@@ -2600,7 +2619,7 @@ void main(void) {
                             for (x = 0; x < 3; x++) {
                                 // Calculate difference of Mains and PV electric meter
                                 if (PVMeter) CM[x] = CM[x] - PV[x];             // CurrentMeter and PV resolution are 1mA
-                                Irms[x] = CM[x]/100;                            // reduce resolution of Irms to 100mA
+                                Irms[x] = (int)CM[x]/100;                       // reduce resolution of Irms to 100mA
```

Comment:
Must be (int)(CM[x]/100), otherwise a current measurement > 32,767 A won't work
Cast (int) has a higher precedence as the division, so signed long CM (eg. 123456mA) is converted into an signed int (32767) first, then divided by 100 (327,67 *0,1A). But it should be 1234,56 *0,1A

Reply:
You're right, I forgot to take that comment from the other PR.